odl/discr/discr_mappings.py

```diff
@@ -63,20 +63,20 @@ def __init__(self, map_type, fset, grid, dspace, order='C', linear=False):
         fset : `FunctionSet`
             The undiscretized (abstract) set of functions to be
             discretized
-        grid :  `TensorGrid`
-            The grid on which to evaluate. Must be contained in
-            the common domain of the function set.
+        partition : `RectPartition`
+            Partition of (a subset of) ``fset.domain`` based on a
```

Comment:
Well that is one test, do all the norm, inner, etc propagate properly etc? Do we have a test for this?